keras_retinanet/utils/anchors.py

```diff
@@ -171,16 +171,20 @@ def bbox_transform(anchors, gt_boxes, mean=None, std=None):
     elif not isinstance(std, np.ndarray):
         raise ValueError('Expected std to be a np.ndarray, list or tuple. Received: {}'.format(type(std)))
 
-    anchor_widths  = anchors[:, 2] - anchors[:, 0] + 1.0
-    anchor_heights = anchors[:, 3] - anchors[:, 1] + 1.0
+    anchor_widths  = anchors[:, 2] - anchors[:, 0]
```

Comment:
I thought we needed these +1.0s due to dataset reasons? Can't fully recall, but I remember being told something along those lines?

Reply:
Those +1's are from `py-faster-rcnn` originally. They encode a box as `[x1, x2]`, `[y1, y2]`, we encode it as `[x1, x2)` and `[y1, y2)`. That means we should compute the width as `x2 - x1` and height as `y2 - y1`. `py-faster-rcnn` (and Detectron for that matter) computes it as `x2 - x1 + 1` and `y2 - y1 + 1`.

Somewhere a while back we switched to this new notation, which in my opinion makes much more sense. However this code was left unchanged.